<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pagina_registro</class>
 <widget class="QMainWindow" name="Pagina_registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Registro</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre(s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_nombre">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellidos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="texto_apellidos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="telefono">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="correo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Correo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefono</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Verificar  contraseña</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registro">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrarme</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>¿Ya tienes una cuenta? </string>
    </property>
   </widget>
   <widget class="QPushButton" name="inicio_sesion">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicia sesión</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
